packages/dds/tree/src/feature-libraries/editable-tree/utilities.ts

```diff
@@ -12,11 +12,11 @@ import { fieldKinds } from "../defaultFieldKinds";
 import { FieldKind } from "../modular-schema";
 
 /**
- * @returns true iff `schema` trees should default to being viewed as just their value when possible.
+ * @returns true if `schema` trees should default to being viewed as just their value when possible.
```

Comment:
This was not a typo. I used [iff](https://en.wikipedia.org/wiki/If_and_only_if) on purpose so that it is unambiguous that false means not a primitive.